Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -837,21 +838,27 @@ DataProcessorSpec getTRDGlobalTrackingSpec(bool useMC, GTrackID::mask_t src, boo
 
   // Request PID policy data
   if (withPID) {
+    // request policy
     switch (policy) {
       case PIDPolicy::LQ1D:
-        // inputs.emplace_back("LQ1D", "TRD", "LQ1D", 0, Lifetime::Condition, ccdbParamSpec("TRD/ppPID/LQ1D"));
+        inputs.emplace_back("lq1dlut", "TRD", "LQ1D", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/PID/LQ1D"));
         break;
       case PIDPolicy::LQ3D:
-        // inputs.emplace_back("LQ3D", "TRD", "LQ3D", 0, Lifetime::Condition, ccdbParamSpec("TRD/ppPID/LQ3D"));
+        inputs.emplace_back("lq3dlut", "TRD", "LQ3D", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/PID/LQ3D"));
         break;
-      case PIDPolicy::Test:
-        inputs.emplace_back("mlTest", "TRD", "MLTEST", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/pid/xgb1"));
+#ifdef TRDPID_WITH_ONNX
+      case PIDPolicy::XGB:
+        inputs.emplace_back("xgb", "TRD", "XGB", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/PID_new/xgb"));
         break;
+      case PIDPolicy::PY:
+        inputs.emplace_back("py", "TRD", "py", 0, Lifetime::Condition, ccdbParamSpec("TRD_test/PID_new/py"));
+        break;
+#endif
       case PIDPolicy::Dummy:
         break;
-      default:
-        throw std::runtime_error("Unable to load requested PID policy data!");
```

Comment:
@f3sch since the PID policies that require ONNX are not visible to the compiler in case it is not available you cannot remove the `default` here. It triggers a `Wswitch` and since all warnings are treated as errors the compilation on mac fails